adapters/coap-vertx-base/src/main/java/org/eclipse/hono/adapter/coap/DeviceRegistryBasedCertificateVerifier.java

```diff
@@ -214,46 +212,42 @@ public void setResultHandler(final HandshakeResultHandler resultHandler) {
     }
 
     @Override
-    public List<CertificateType> getSupportedCertificateType() {
+    public List<CertificateType> getSupportedCertificateTypes() {
         return SUPPORTED_TYPES;
     }
 
     @Override
     public CertificateVerificationResult verifyCertificate(
             final ConnectionId cid,
             final ServerNames serverName,
-            final Boolean clientUsage,
-            final boolean truncateCertificatePath,
-            final CertificateMessage message,
-            final DTLSSession session) {
+            final InetSocketAddress remotePeer,
+            final boolean clientUsage,
+            final boolean verifySubject,
+            final boolean truncateCertificatePath, final CertificateMessage message) {
```

Comment:
can you please put the last parameter on its own line as well?
*verifySubject* and *truncateCertificatePath* are not used, or am I mistaken?

Reply:
done